src/radical/pilot/configs/resource_xsede.json

```diff
@@ -1,5 +1,49 @@
 
 {
+    "expanse": {
+        "description"                 : "(https://www.sdsc.edu/support/user_guides/expanse.html).",
+        "notes"                       : "Always set the ``project`` attribute in the PilotDescription.",
+        "schemas"                     : ["local", "ssh"],
+      # "mandatory_args"              : [],
+        "local"                      :
+        {
+            "job_manager_endpoint"    : "slurm://expanse.sdsc.xsede.org",
+            "filesystem_endpoint"     : "file://expanse.sdsc.xsede.org"
+        },
+        "ssh"                         :
+        {
+            "job_manager_endpoint"    : "slurm+ssh://expanse.sdsc.xsede.org",
+            "filesystem_endpoint"     : "sftp://expanse.sdsc.xsede.org"
+        },
+        "default_queue"               : "compute",
+        "resource_manager"            : "SLURM",
+        "lfs_path_per_node"           : "${LOCAL}",
+        "lfs_size_per_node"           : 1000000,
+        "cores_per_node"              : 128,
+        "gpus_per_node"               : 0,
+        "agent_config"                : "default",
+        "agent_scheduler"             : "CONTINUOUS",
+        "agent_spawner"               : "POPEN",
+        "launch_methods"              : {
+                                         "order" : ["FORK","MPIRUN"],
```

Comment:
You probably want to remove `FORK` as that would not be able to place tasks on other nodes.

Reply:
Great point, sorry missed that.

Reply:
@andre-merzky  Done.